dali/test/python/test_RN50_data_fw_iterators.py

```diff
@@ -152,6 +133,22 @@ def test_fw_iter(IteratorClass, args):
             iters += 1
 
     if iterator_name == "nvidia.dali.plugin.tf.DALIIterator":
+        import tensorflow as tf
```

Comment:
how about moving this to `import_tf` ?

Reply:
In L1154 we are using tf as a name and it need to be known in this scope.

Reply:
After some consideration I have followed your suggestion